packages/kernel-6.12/config-bottlerocket

```diff
@@ -48,6 +48,17 @@ CONFIG_FB=y
 CONFIG_FB_EFI=y
 CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
 
+# kernel-hardening-checker recommendations
+CONFIG_ARCH_MMAP_RND_BITS=33
+CONFIG_ARCH_MMAP_RND_COMPAT_BITS=16
+# CONFIG_BLK_DEV_WRITE_MOUNTED is not set
+# CONFIG_HIBERNATION is not set
+# CONFIG_LEGACY_TIOCSTI is not set
+CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=176
+CONFIG_RANDOMIZE_BASE=y
+CONFIG_RANDOMIZE_KSTACK_OFFSET_DEFAULT=y
+CONFIG_X86_USER_SHADOW_STACK=y
```

Comment:
As I mentioned offline - this doesn't make sense to enable until userspace binaries are built with CET support.

Reply:
This would allow binaries in a container to take advantage of this CET feature, but certainly we can defer this.